examples/social-network/websocket-server/asyncapi.yaml

```diff
@@ -1,37 +1,53 @@
-asyncapi: 2.2.0
+asyncapi: 3.0.0
 info:
   title: The Social Network
   version: 0.1.0
-
 servers:
   websockets:
-    url: ws://0.0.0.0:3001
+    host: '0.0.0.0:3001'
     protocol: ws
   mosquitto:
-    url: mqtt://test.mosquitto.org
+    host: test.mosquitto.org
     protocol: mqtt
     bindings:
       mqtt:
         clientId: the-social-network
-
 channels:
   /:
-    publish:
-      operationId: onLikeDislike
-      message:
+    address: /
+    messages:
+      onLikeDislike.message:
         $ref: '#/components/messages/likeOrDislike'
-    subscribe:
-      message:
+      subscribe.message:
         $ref: '#/components/messages/likeCountUpdated'
     servers:
-      - websockets
+      - $ref: '#/servers/websockets'
   post/liked:
-    subscribe:
-      message:
+    address: post/liked
+    messages:
+      subscribe.message:
         $ref: '#/components/messages/notifyPostLiked'
     servers:
-      - mosquitto
-
+      - $ref: '#/servers/mosquitto'
+operations:
+  onLikeDislike:
+    action: receive
+    channel:
+      $ref: '#/channels/~1'
+    messages:
+      - $ref: '#/components/messages/likeOrDislike'
+  /.subscribe:
+    action: send
+    channel:
+      $ref: '#/channels/~1'
+    messages:
+      - $ref: '#/components/messages/likeCountUpdated'
+  post/liked.subscribe:
+    action: send
+    channel:
+      $ref: '#/channels/post~1liked'
+    messages:
+      - $ref: '#/components/messages/notifyPostLiked'
```

Comment:
Please carefully review the operation Ids here. Avoid pub/sub, slashes, and dots as much as possible.